src/test/java/io/vertx/test/core/VertxTestBase.java

```diff
@@ -42,6 +42,9 @@
   @Rule
   public RepeatRule repeatRule = new RepeatRule();
 
+  @Rule
+  public FileDescriptorLeakRule fileDescriptorLeakRule = new FileDescriptorLeakRule();
```

Comment:
I believe that the vertx test base should not annotated by default because it will cause some overhead in testing. Instead we should have an FD leak specific test that will test a few operations we have like starting HTTP server, using file system, etc... and we will have of course the HTTP/2 send file test.

Reply:
Overhead is added only when the annotation is present in one / many test methods

If we want to create separate tests just for this then for each test we have to setup 
-vertx
-http
-ssl if http2 
and so on

With these method everything is already there since its run on top of the existing tests

Reply:
ah right, so it's good here.

I think it's fine like this then. We'll see later if we need to establish a specific test class.